variables.tf

```diff
@@ -20,6 +20,18 @@ variable "instance_profile" {
   default     = null
 }
 
+variable "metadata_http_tokens_required" {
```

Comment:
We usually use an `_enabled` naming convention

```suggestion
variable "metadata_http_tokens_enabled" {
```

Reply:
See https://github.com/cloudposse/terraform-aws-ec2-instance/blob/master/docs/terraform.md#input_metadata_http_tokens_required

I want to be consistent in two modules. Let me know if you want to fix both then

Reply:
I think initially it was because the resulting option is "required", so looks like just an oversight in the ec2 module

Reply:
Ah ok that makes sense then I wouldn't change it. Let's keep it consistent.

Reply:
Decided to not change to keep both consistent and backward compatible